src/css/object/project/job-search.scss

```
@@ -0,0 +1,74 @@
+@use '../../object/utility/utility.scss';
+
+.p-jobsearch {
+  Width:1181px;
+  Height:239px;
+  border-radius: 10px 10px 10px 10px;
+  border: solid 3px #DEEEF9;
+  background-color: #DEEEF9;
+  margin:auto;
+  text-align: center;
+  line-height:44px;
```

半角スペース開けましょう
こういうところ他にもないか見直しお願いします！
```suggestion
  line-height: 44px;
```

---

修正しました。
7ccff574e47682254a0ee120272d774be9320968